docs/includes/generated_docs/schemas/beta.tpp.md

```diff
@@ -2042,7 +2076,16 @@ Patient's date of death. Only deaths registered from February 2019 are recorded.
 <p class="dimension-indicator"><code>many rows per patient</code></p>
 ## opa
 
+Outpatient appointments data (OPA) is provided via the NHS Secondary Uses Service.
+
+This table gives core details of outpatient appointments.
+
+Refer to the GitHub issue that [describes limitations
+of the outpatient appointments data][opa_limitations_issue]
+and the GitHub issue that [discusses more of the background context][opa_context_issue].
 
+[opa_limitations_issue]: https://github.com/opensafely-core/cohort-extractor/issues/673
+[opa_context_issue]: https://github.com/opensafely-core/cohort-extractor/issues/492
```

Comment:
question: As the APCS reference information, should we only include the reference links in the `opa` table, as has been done, or duplicate it in each OPA-related table?